front/src/common/ScenarioExplorer/ScenarioExplorerModalProjectMiniCard.tsx

```diff
@@ -9,17 +10,14 @@ export default function ProjectMiniCard({
 }: MiniCardsProjectProps) {
   return (
     <div
-      className={`scenario-explorator-modal-part-itemslist-minicard project with-image ${
-        isSelected ? 'selected' : ''
-      } ${project.studies_count && project.studies_count > 0 ? '' : 'empty'}`}
+      className={cx(
+        'scenario-explorator-modal-part-itemslist-minicard project with-image',
+        isSelected && 'selected',
+        project.studies_count === 0 && 'empty'
+      )}
       role="button"
       tabIndex={0}
-      onClick={() =>
-        !isSelected &&
-        project.studies_count &&
-        project.studies_count > 0 &&
-        setSelectedID(project.id)
-      }
+      onClick={() => !isSelected && project.studies_count > 0 && setSelectedID(project.id)}
```

Comment:
We do not care about the value returned by this arrow function, just that it’s calling another function. 

A more imperative looking syntax makes the intent more obvious at first sight.

```suggestion
      onClick={() => {
        if(!isSelected && project.studies_count > 0) {
          setSelectedID(project.id)
        }
      }
```

Reply:
Done thanks !